Core/DISET/RequestHandler.py

```diff
@@ -78,6 +79,24 @@ def _rh__initializeClass( cls, serviceInfoDict, lockManager, msgBroker, monitor
     cls.__monitor = monitor
     cls.log = gLogger
 
+    allNames = dir( cls )
+    exportedMethods = [ name for name in allNames if 'export_' in name ]
+    system, service = cls.__svcName.split( '/' )
+    setup = gConfigurationData.extractOptionFromCFG( 'DIRAC/Setup' )
+    systemSetup = gConfigurationData.extractOptionFromCFG( 'DIRAC/Setups/%s/%s' % ( setup, system ) )
+    result = gConfigurationData.extractOptionFromCFG( '/Systems/%s/%s/Services/%s/ResponseMonitoring/Methods' % ( system, systemSetup, service ) )
```

Comment:
gConfigurationData is not declared. I do not know how this is tested. Anyway I applied your recommendation.